proposals/2018/Rename Root Element.md

```diff
@@ -15,7 +15,7 @@ Standard 211-type audit. For example:
 4) transferring building data for use in M&V IPMVP Option C/D type analysis
```

Comment:
We don't want to update the proposal from 2018/Rename Root Element. This should be a new proposal in 2019 for External References.